tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -684,3 +700,80 @@ def sync_volumes(volumes, index, vol_options, sync_ttl):
             b_index += 1
         except KeyError:
             break
+
+
+def brick_status_alert(hostname):
+    # fetching brick details of disconnected node
+    bricks = NS._int.client.read(
+        "clusters/%s/Bricks/all/%s" % (
+            NS.tendrl_context.integration_id,
+            hostname
```

Comment:
cant you use NS.gluster.objects.Brick.load_all()  ?

Reply:
We can i am not sure how it will behave for lock, i will try after i comment